### DISCUSSION TITLE
How to change the style of Slider?

### DISCUSSION THREAD

**Discussion by alessandrofrancesconi** [AvaloniaUI/Avalonia#5816] [Q&A] [answered]
I would like to change the style of the [Slider](https://avaloniaui.net/docs/controls/slider). In particular I would like to:

- reduce the size of the thumb
- change the color of the thumb
- reduce the height (or padding?) of the control 

If useful, I'm using the Fluent Theme

**Comment by maxkatz6** [comment, marked as answer]
Use slider styles code as a reference 
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Fluent/Controls/Slider.xaml

You can define your own styles, which will override default ones. Or you can create style with new whole slider ControlTemplate. 
Also pay attention on resources, some of them can be redefined without styles change.

**Reply by alessandrofrancesconi** [reply]
Thanks I'll try that
